<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewLib</class>
 <widget class="QDialog" name="viewLib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>词库</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>词库管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="personalL">
        <property name="text">
         <string>个人词库</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="wrongL">
        <property name="text">
         <string>错词库</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="defaultL">
        <property name="text">
         <string>系统词库</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>单词</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>词性</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>释义</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="AddBtn">
        <property name="text">
         <string>新增</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ModifyBtn">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteBtn">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearBtn">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
